scanner/mapping.go

```diff
@@ -130,6 +130,15 @@ func (s mediaFileMapper) trackID(md metadata.Tags) string {
 
 func (s mediaFileMapper) albumID(md metadata.Tags) string {
 	albumPath := strings.ToLower(fmt.Sprintf("%s\\%s", s.mapAlbumArtistName(md), s.mapAlbumName(md)))
+
+	if !conf.Server.Scanner.GroupAlbumEditions {
+		// don't really like this - using mapDates() means parsing all three date tags again, seems redundant
```

Comment:
Why not move this logic up in toMediaFile, where you can either call the default function, or one with the release date passed in?